docker-compose.yml

```diff
@@ -34,3 +44,11 @@ elasticsearch:
 
 redis:
   image: redis
+
+rabbitmq:
+  image: rabbitmq:3-management
+  hostname: olympia
+  name: olympia
+  ports:
+    - "5672:5672"
```

Comment:
Turns out that if you run `docker-compose` once with this as `ports` instead of `expose`, you have to manually delete the container before it will notice that you switched it to `expose`... In any case, with `ports`, this conflicts with my local RabbitMQ and explodes. With `expose` it works.